organise_desktop/Clean.py

```diff
@@ -4,19 +4,25 @@
 from .cronController import  schedule_end, schedule_start
 from .organiseDesktop import undo, organise_desktop
 
-if sys.version_info >= (3,):
+try:  # This will throw an error in python 2.x
     from tkinter import *
     from tkinter import messagebox as tkMessageBox
-else:
-    from tkinter import *
+except ImportError:
+    from Tkinter import *
     import tkMessageBox
 
 pwd = os.path.dirname(os.path.abspath(__file__))
 Extensions = json.load(open(pwd+'/Extension.json', 'r'))
-folders = [x for x in Extensions]
 
 class App(Frame):
     """define the GUI"""
+
+    def __init__(self, master=None):
+        Frame.__init__(self, master)
+        self.pack()
+        self.create()
+        self.folders = [x for x in Extensions]
```

Comment:
This is a good initiative, but you would need to change all the `folders` variables to `self.folder` otherwise it will not use `self.folder` at all.